_data/categories.yml

```diff
@@ -2,37 +2,45 @@
 #   slug: code-generators
 
 - name: Converters
+  description: Various tools to convert to and from OpenAPI standards. Useful for working with various API formats.
   slug: converters
 
 - name: Documentation
+  description: Generators to help build API documentation based off a variety of formats and standards.
   slug: documentation
 
 - name: Text Editors
-  slug: text-editors
```

Comment:
You would like all slugs removed?

Reply:
No I want you to not accidentally remove the slugs, :)

Click "Files Changed" on this pull request, and if anything looks strange (like you accidentally changing "text-editors" to "editors", which you probably didnt do intentionally) then update the commits in your PR to avoid those changes.